pkgs/applications/science/electronics/verilog/default.nix

```diff
@@ -1,14 +1,22 @@
-{stdenv, fetchurl, gperf, flex, bison}:
+{ stdenv, fetchFromGitHub, pkgs }:
```

Comment:
-1 on using the whole package set ("pkgs"). It is cleaner, although a bit more verbose, to specify each dependency by itself.